README.rst

```diff
@@ -62,20 +62,17 @@ Development Setup
 To setup this project for development do the following
 
 1. Fork the repository.
-2. Clone the repository locally using ``git clone``.
+2. Clone the repository locally using ``git clone https://github.com/xxxxxx/netjsonconfig-editor.js.git``.
```

Comment:
use `https://github.com/<your_fork>/netjsonconfig-editor.js.git` for consistency with other OpenWISP repositories